marlowe/debugging-cookbook.md

```diff
@@ -1,16 +1,22 @@
 # The Marlowe Debugging Cookbook
 
-*   [General approaches](#general-approaches)
-    *   [Intrastructure](#infrastructure)
-    *   [Marlowe semantics](#marlowe-semantics)
-    *   [Time intervals](#time-intervals)
-    *   [Transactions](#transactions)
-    *   [Marlowe validator script](#marlowe-validator-script)
-*   [Specific approaches](#specific-approaches)
-    *   [TEApplyNoMatchError](#reapplynomatcherror)
-    *   [TEAmbiguousTimeIntervalError](#teambiguoustimeintervalerror)
-*   [Debugging Plutus](#debugging-plutus)
-*   [Profiling Marlowe](#profiling-marlowe)
+- [The Marlowe Debugging Cookbook](#the-marlowe-debugging-cookbook)
```

Comment:
This was autogenerated by the `Markdown All in One` plugin. I can revert if you don't like

Reply:
Seems like we should suppress the generation of `h1` links for titles like this or use different markup for titles. Perhaps @sfoo-iohk has a suggestion.